From: Dermot Bradley <dermot_bradley@yahoo.com>
Date: Thu, 4 Sep 2020 14:51 +0100
Subject: [PATCH] cloud-init: Prevent debug output line appearing on console

During boot a debug lines appears on the console, this appears to be been
mistakenly added to the upstream code.

This patch removes the debug line. This change has already been applied
upstream, therefore this patch can be removed upon the next upstream release.

---

--- a/cloudinit/util.py
+++ b/cloudinit/util.py
@@ -2490,7 +2490,6 @@
         LOG.debug("dmidata is not supported on %s", uname_arch)
         return None
 
-    print("hi, now its: %s\n", subp)
     dmidecode_path = subp.which('dmidecode')
     if dmidecode_path:
         return _call_dmidecode(key, dmidecode_path)
